### ISSUE TITLE
Allow using JSON directly in MessagePactBuilder

### ISSUE THREAD

**Issue by JamesFraser-FD** [pact-foundation/pact-jvm#1669]
All examples I can see for `MessagePactBuilder` require constructing a `DslPart` object to pass into the `.withContent()` method

Our current project has existing JSON files and it would be nice if we could pass in this data directly, either as a String or JSONObject - similar to how `ConsumerPactBuilder` does already ([example](https://docs.pact.io/implementation_guides/jvm/consumer#dsl-usage))

Something similar to this or the below sample would be ideal:
```java
JSONObject jsonObject = loadJsonFromFile(...);

return new MessagePactBuilder()
    .expectsToReceive("a user created message")    
    .withContent(jsonObject)
    .toPact();
```

**Comment by rholshausen** [comment]
Looks like support for String data was implemented for #1619

**Comment by rholshausen** [comment]
4.4.7 released
